webapp/src/webapp/events/hoop_app.cljs

```diff
@@ -40,14 +23,40 @@
  (fn [{:keys [db]} [_ response]]
    {:db (assoc-in db [:hoop-app->my-configs] response)}))
 
+(rf/reg-event-fx
+ ::hoop-app->start
```

Comment:
are we going with private events now? I have no problems with this approach, just wanted to check where we are going and what's the plan

Reply:
Thanks, I let it slip by without realizing it.